libs/utils/utils.js

```diff
@@ -151,7 +151,8 @@ export const [setConfig, getConfig] = (() => {
       config.autoBlocks = conf.autoBlocks ? [...AUTO_BLOCKS, ...conf.autoBlocks] : AUTO_BLOCKS;
       document.documentElement.setAttribute('lang', config.locale.ietf);
       try {
-        document.documentElement.setAttribute('dir', (new Intl.Locale(config.locale.ietf)).textInfo.direction);
+        const contentDir = getMetadata('content-direction');
+        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || (new Intl.Locale(config.locale.ietf)).textInfo.direction);
```

Comment:
```suggestion
        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || 'ltr';
```

Reply:
@mokimo @TsayAdobe I have made the requested changes. Could you pelase review